packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -1016,10 +1028,13 @@ export function useGlobalStylesOutput() {
 			return [];
 		}
 		mergedConfig = updateConfigWithSeparator( mergedConfig );
+
 		const blockSelectors = getBlockSelectors(
 			getBlockTypes(),
 			getBlockStyles
 		);
+		const blockNames = getBlockNames( getBlockTypes() );
```

Comment:
Suggestion (non-blocking): This could be "simplified" to something like this:

```js
getBlockTypes().map( ( blockType ) => blockType.name );
```

Then we can directly use the array below to push block stylesheets. No need for `Object.entries( blockNames )`.

Reply:
good catch! Done 👍

Reply:
It looks like you had a better idea than me in [ccdc5df](https://github.com/WordPress/gutenberg/pull/46571/commits/ccdc5df59f66f39fb6948b34e690c03b3b86b3cf). You can also drop the `.map()` and loop over the block object directly.